pkg/gadgets/oomkill/types/types.go

```diff
@@ -21,10 +21,10 @@ import (
 type Event struct {
 	eventtypes.Event
 
-	TriggeredPid  uint32 `json:"tpid,omitempty"`
-	TriggeredComm string `json:"tcomm,omitempty"`
-	KilledPid     uint32 `json:"kpid,omitempty"`
-	KilledComm    string `json:"kcomm,omitempty"`
+	TriggeredPid  uint32 `json:"fpid,omitempty"`
```

Comment:
What does `fpid` stands for?

Reply:
To be honest: I do not know.
The BCC code uses `fpid` and `tpid`. I think `t` is used for target but for `f` I do not know.
Since I did not know the meaning of `f` I used other letters in the initial eBPF code I wrote.

I modified this when adding the upstream one but I think this modifications is not needed, thus I will revert changes to this file.